tester.js

```diff
@@ -9,60 +9,23 @@ function log(message) {
     return process.stdout.write(`${message}\n`);
 }
 
-function installDependency(dir) {
-    log(`installing dependencies ${dir}`);
-
-    try {
-        fs.statSync(path.join(dir, 'package.json'));
-    } catch (e) {
-        return log('package.json file not found, skip installation');
-    }
+function testing(dir) {
```

Comment:
do we need to have tester.js and the separate package.json's at all?  Would it be possible to just have the base package.json and then do a call similar to what is done in Vault:
 "ft_test": "mocha --timeout 8000 --recursive --require ./es6-compat.js tests/functional",